core/src/edu/asu/c3simulator/screens/BusinessCreationGuide.java

```diff
@@ -18,7 +19,7 @@ public BusinessCreationGuide(Game game)
 	{
 		resetAllGuideScreens();
 		//TODO add Industry screen
-		game.setScreen(new IndustryScreen(game));
+		game.setScreen(AllManagementScreens.INDUSTRY.getInstance());
```

Comment:
Not as clean as I'd like it to be, but better than before. Maybe we can add a method to C3Simulator that takes a ManagementScreens enumeration...